src/ProjectFileTools/Exports/ExportedWorkspaceManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Utilities;
+using ProjectFileTools.MSBuild;
+
+namespace ProjectFileTools.Exports
+{
+    [Export(typeof(WorkspaceManager))]
+    [Name("Default WorkspaceManager")]
+    public class ExportedWorkspaceManager : WorkspaceManager
```

Comment:
Why is this a separate class? You can put the export on the `WorkspaceManager` directly.